### ISSUE TITLE
QUESTION:  Status of this Project

### ISSUE THREAD

**Issue by mprimeaux** [apache/cassandra-gocql-driver#1749]
Hi all. 

Please don’t interpret this question as anything other than constructive as I’m working to understand the current Go-based client library space for Apache Cassandra.

I see the last commit to this popular library was [nine months ago](https://github.com/gocql/gocql/commit/34fdeebefcbf183ed7f916f931aa0586fdaa1b40) as of this post. Granted, currently, the latest version of Cassandra is [4.1.x](https://github.com/apache/cassandra/tags) with `5.x` beta pending and this library does support Cassandra [4.1.x](https://github.com/gocql/gocql?tab=readme-ov-file#supported-versions).

The reason for my inquiry is to understand why the last commit was nine months ago. I completely understand if the client is stable in relation to Cassandra’s API surface but it seems the accrual technical risk related to potential CVEs and technical debt related to the supply chain would warrant more recent updates.

Again, please don’t interpret this as anything other than constructive. We have quite a few services written in Go and are considering Cassandra for an area of our system that would benefit from non-relational columnar-orientated durable storage and are looking to find the “best” Go client to use. 

Your help, insights, and guidance are greatly appreciated.

**Comment by martin-sucha** [comment]
Hello!

you can find some information [in the mailing list thread](https://groups.google.com/g/gocql/c/v0FruczBb2w/m/KBso2NfJAQAJ).

TL;DR: This project needs new maintainers. There are a few people who are keen to contribute to this project. There is also a [discussion whether the Cassandra project should adopt the driver](https://lists.apache.org/thread/h8q3lgqgsb87vptxr1j6x322yqz9fk7l).

**Comment by joao-r-reis** [comment]
This project has been donated to the Apache Software Foundation and a new group of committers and contributors are slowly ramping up on the project. Feel free to reach us on the [Apache Cassandra mailing list](https://cassandra.apache.org/_/community.html) or on the [#cassandra-drivers channel on the ASF slack ](https://the-asf.slack.com/archives/C05LPRVNZV1) if you have any more questions.
